restli-server/src/test/java/com/linkedin/restli/server/test/TestRestLiRouting.java

```diff
@@ -94,7 +95,7 @@
   public void testRoutingDetailsCollectionGet(ProtocolVersion version, String uri) throws Exception
   {
     Map<String, ResourceModel> pathRootResourceMap = buildResourceModels(StatusCollectionResource.class);
-    _router = new RestLiRouter(pathRootResourceMap);
+    _router = new RestLiRouter(pathRootResourceMap, new RestLiConfig());
```

Comment:
Alternatively, if you keep the existing constructor (which you definitely should) but choose to not deprecate it, then you can use that here instead of adding this needless empty config everywhere in this file.